charts/camunda-platform/values.yaml

```diff
@@ -1746,6 +1746,105 @@ web-modeler:
       # Postgresql.auth.database defines the name of the database to be created for Web Modeler
       database: web-modeler
 
+# Connectors configuration for the Connectors.
+connectors:
+  # Enabled if true, the Connectors deployment and its related resources are deployed via a helm release
+  enabled: true
```

Comment:
Connectors should be disabled by default (once the tests are done).
And we need to enable it in the [integration tests](https://github.com/camunda/camunda-platform-helm/tree/4c6de38/test/integration/scenarios), namely in:
- chart-with-custom-values
- chart-with-keycloak-v19
- chart-with-web-modeler

Reply:
This MVP doesn't require Keycloak. As such, we discussed it today that we should enable connectors by default. Or do you have concerns with that @aabouzaid?

cc: @bastiankoerber

Reply:
Will the Connectors be used by all users (or at least the majority of the users) by default?
Based on my understanding it's an extension used when the users need to extend our platform.
That's why don't think it should be enabled by default.

Reply:
Connectors are the core feature, so we assume that major amount of customers will be using them.
---
Connectors are not using Keycloak just yet, but will be used as a result of https://github.com/camunda/camunda-platform-helm/issues/581, which is one of our next milestones. We tentatively targeting 23Q1 for it.
---
Not sure what you mean by:

> then we only need it in chart-with-custom-values and chart-with-web-modeler

Do you mean that we just need to include the:
```
connectors:
  enabled: true
```

to the `values-custom.yaml` or something else?

Reply:
@igpetrov for 8.1.x we should not introduce a new component that could affect the default behaviour, so for sure it will be disabled.
For 8.2 I've asked the PM about it since I don't really know if that would be used by everyone. If it will be added, then we need to add it to the v8.2 checklist: https://github.com/camunda/camunda-platform-helm/issues/543

regarding to the tests, you need to add them to the tests values file:
- [values-web-modeler-enabled.yaml](https://github.com/camunda/camunda-platform-helm/blob/e3e1c3d27eb6d7cacfd3368e76663abce3089bc6/test/integration/scenarios/chart-with-web-modeler/values-web-modeler-enabled.yaml#L1)
- [values-custom.yaml](https://github.com/camunda/camunda-platform-helm/blob/e3e1c3d27eb6d7cacfd3368e76663abce3089bc6/test/integration/scenarios/chart-with-custom-values/values-custom.yaml#L1)

Reply:
So if @bastiankoerber confirmed it will be enabled by default in 8.2 (the next minor release) then please add it to https://github.com/camunda/camunda-platform-helm/issues/543)

Reply:
Also added to the ticket

Reply:
It's still `enabled: true`.
https://github.com/camunda/camunda-platform-helm/blob/ca6fadd4c49f52d611a2fc163f9c44250f21d444/charts/camunda-platform/values.yaml#L1752